gnt4/docs/guides/opcode_group/47.md

```diff
@@ -1,7 +1,74 @@
 # Opcode Group 47
 
-## 4707
-Set projectile DIR and ANG
-Uses SEQ_RegCMD2 to get the arguments
-Example where XXXX is DIR and YYYY is ANG:
-47073F00 0000XXXX 3F000000 0000YYYY
+## 4700 spawn\_projectile
```

Comment:
Are the `\` escapes needed for the `_` here?

Reply:
Probably not strictly necessary

Reply:
Oh interesting, I forgot that markdown uses `_` surrounding text for italics, so it probably is necessary to be escaped when there are multiple `_` in a header. I was just wondering why I didn't need to escape them in https://github.com/NicholasMoser/Naruto-GNT-Modding/blob/master/gnt4/docs/guides/opcode_group/04.md#0406---i32_dec (which is only because each mnemonic only has a single `_`)